HYDRA_PROD_VER

```diff
@@ -1 +1 @@
-bfe743291af35f497069092a8c9e454b6bca6f18
+f86fc7a197081df6468784a30c7d002232665ceb
```

Comment:
what does the sha point to?

Reply:
it's a commit hash! specifically: https://github.com/hydradatabase/hydra/commit/f86fc7a197081df6468784a30c7d002232665ceb

in practice, it's used for the upgrade tests.